deploy/servicemesh-operator.yaml

```diff
@@ -127,23 +127,27 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - pods
-  - services
+  - configmaps
   - endpoints
+  - namespaces
   - persistentvolumeclaims
-  - events
-  - configmaps
+  - pods
+  - replicationcontrollers
   - secrets
   - serviceaccounts
-  - namespaces
-  - routes
```

Comment:
is routes still needed?

Reply:
It's namespaced now, route.openshift.io/v1 instead of just v1.